### PR TITLE
Update README to reflect the need for a response.

### LINKED ISSUE #2: Example does not respond?
Unless I’m mistaken, your example returns nothing (and thus would throw in CF land). event.waitUntil(Promise) resolves *after* you've returned a Response, so you'd still need to return something (a Response) after that line to hand the actual Response to the request. 😁

### PR DESCRIPTION
**Pull request by kwhitley**
Just added a response to the catch block while the `event.waitUntil` fires behind the scenes...

Fixes #2.

### CODE REVIEW

**Comment by kwhitley** [comment]
Fixes #2

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ const sentry = new Sentry({ dsn: "https://123@456.ingest.sentry.io/789" });
 addEventListener("fetch", (event) => {
   event.respondWith(
     handler(event.request).catch((err) => {
-      // Wait until the response from Sentry has resolved.
+      // Wait until the response from Sentry has resolved (will continue after return below)
```
Comment (blakeembrey) [inline README.md]: Just a heads up, but I'm certain the implementation has nothing to do with the return below. I could test it but I believe that `event.waitUntil` is only keeping the worker open until the promise resolves, not waiting for a return.

Comment (blakeembrey) [reply README.md]: I added some language based on the documentation here: https://developers.cloudflare.com/workers/runtime-apis/fetch-event#methods. Specifically around "extending the event lifetime". Let me know if the changes sound good to you.

Comment (kwhitley) [reply README.md]: Yep that's much better wording.  👍 

I meant it as "this function/promise will continue working after the Response return below", but obv that's more confusing (and roundabout) than yours 😄

Comment (blakeembrey) [reply README.md]: Gotcha, makes sense 👍 Thanks!

Comment (kwhitley) [reply README.md]: 🥳

**Pull request merged**